src/main/resources/db/migration/V4_7__create_table_motedeltaker_arbeidsgiver_varsel.sql

```diff
@@ -0,0 +1,23 @@
+-- ROLLBACK-START
+------------------
+-- DROP TABLE MOTEDELTAKER_ARBEIDSGIVER_VARSEL;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSGIVER_VARSEL_MD_AT_ID;
```

Comment:
Hvis dette skal brukes til noe bør rekkefølgen endres, men dette er kanskje mest til informasjon om hva som må gjøres for å reversere?

Reply:
Endret.